model/src/main/kotlin/licenses/LicenseInfoResolver.kt

```diff
@@ -107,6 +107,10 @@ class LicenseInfoResolver(
             license.builder().apply {
                 sources += LicenseSource.DETECTED
                 resolvedLocations[license]?.let { locations.addAll(it) }
+
+                licenseInfo.detectedLicenseInfo.findings.forEach { findings ->
+                    originalExpressions[LicenseSource.DETECTED] = findings.licenses.map { it.license }.toSet()
```

Comment:
Nit: Could this be `mapTo(mutableSetOf())`? Note that due to the assignment the mutable set is only seen as an immutable set when used.